blazor/rating/tooltip.md

```diff
@@ -45,11 +41,9 @@ Below example demonstrates the tooltip template of Rating.
 
 ## Tooltip customization
 
-The Rating Tooltip can be customized by using the CssClass property, which accepts custom CSS class names that defines the specific user-defined styles and themes to be applied on the Tooltip element.
-
-N> You can also explore more about [Tooltip Customization](https://blazor.syncfusion.com/documentation/tooltip/style) that shows how to customize styles and appearance of tooltip.
+You can customize the appearance of the tooltips using the `CssClass` property of the Blazor Rating component to specify custom CSS class names that define user-defined styles and themes to be applied to the tooltip element.
 
-Below example demonstrates the tooltip customization of Rating.
+N> You can find more information about customizing the appearance of the tooltip in the [Tooltip Customization](https://blazor.syncfusion.com/documentation/tooltip/style) documentation.
```

Comment:
Confirm this

Reply:
I have confirmed the above mentioned and there is no issue.